table/column.go

```diff
@@ -10,7 +10,8 @@ type Column struct {
 	Key   string
 	Width int
 
-	style lipgloss.Style
+	Filterable bool
```

Comment:
I would prefer not to export this, since `WithFiltered` then handles it when chained with `NewColumn`.